bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -49,13 +49,18 @@
            }
          ],
          "outputs":[
-            {
+           {
                "name":"command",
                "type":"command",
                "description":"the received command",
                "label":"Command"
-            }
-          ]
+           },
+           {
+                "name":"event",
```

Comment:
Note that we had such an output previously, see https://github.com/smerschjohann/smarthome/blob/bddea50c1c6df2dd06930abefec52bd86d323e1f/bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/EventTriggersTypeDefinition.json#L36.

If you want to bring it back, I'd suggest to also add the label again and define the type as the FQCN and not merely as "Object".